src/font_face.rs

```diff
@@ -76,7 +87,7 @@ impl FontFace {
         simulations: DWRITE_FONT_SIMULATIONS,
     ) -> FontFace {
         unsafe {
-            let file_ptrs = self.get_raw_files();
+            let file_ptrs = self.raw_files().unwrap();
```

Comment:
Similar to #64, there is no obvious name change here for a non-panicking method and it can be changed in the next API-breaking release